extensions/stackdriver/metric/record.cc

```diff
@@ -28,7 +28,7 @@ namespace Metric {
 void record(bool is_outbound, const ::wasm::common::NodeInfo &local_node_info,
             const ::wasm::common::NodeInfo &peer_node_info,
             const ::Wasm::Common::RequestInfo &request_info) {
-  double latency_ms = request_info.duration / absl::Nanoseconds(1) / 1000.0;
+  double latency_ms = request_info.duration / absl::Milliseconds(1);
```

Comment:
This is a bug that is not about stackdriver filter. or STS Please submit this to master.